pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting.md

```diff
@@ -0,0 +1,275 @@
+---
+$title: Formatting guides & tutorials 
+$order: 3
+description: 'File formatting requirements for amp.dev'
+formats:
+  - websites
+  - stories
+  - ads
+  - email
+author: CrystalOnScript
+---
+
+Guides and tutorials are submitted in [Markdown](https://www.markdownguide.org/), with an additional frontmatter and shortcode formatting. 
```

Comment:
Maybe makes sense to already point out here that certain types of markdown syntax aren't used,  namely I think:

- the markdown image syntax (we have shortcodes, and <amp-img> instead)
- the link syntax (at least for inbound links on amp.dev)

From there you can link to the section below describing the link syntax, then?

Reply:
@CrystalOnScript friendly ping

Reply:
Ah, yup, this got lost - thanks!

Reply:
Added image syntax. Link syntax was included.